### PR TITLE
Update readme documentation for update and delete

### PR DESCRIPTION
**Pull request by jwswj**
@miogalang @joseconsador further readme updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,13 +69,13 @@ $newTicket = $client->tickets()->create([
...
-$client->ticket([123, 456])->update([
-    'status' => 'urgent'
+// Update a ticket
+$client->tickets()->update(123,[
+    'priority' => 'high'
 ]);
 
 // Delete a ticket
-$client->ticket(123)->delete();
+$client->tickets()->delete(123);
```
Comment (miogalang) [inline README.md]: Both would work

**Comment by miogalang** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (jwswj) [inline README.md]: `$client->ticket(123);` throws:

```
Fatal error: Uncaught exception 'Exception' with message 'No method called ticket available in Zendesk\API\HttpClient' in /Users/jsmale/Code/other/testphpapi/vendor/zendesk/zendesk_api_client_php/src/Zendesk/API/Traits/Utility/InstantiatorTrait.php:33
```

Comment (miogalang) [inline README.md]: Ahh my mistake. `$client->tickets(123)->delete();` would also be allowed

Comment (jwswj) [inline README.md]: Sweet

**Pull request merged**